src/WebformCivicrmPostProcess.php

```diff
@@ -2148,6 +2141,9 @@ private function contributionParams() {
         'domain_id' => $domain,
       ));
     }
+    if (empty($params['payment_instrument_id']) && !empty($params['payment_processor_id'])) {
```

Comment:
included the commit from https://github.com/colemanw/webform_civicrm/pull/289 which was a fix for the regression caused by #180

Reply:
@jitendrapurohit - can you please add a few lines to the `ContributionIatsTest` -> section line 180-186 to check that with this PR the `payment_instrument_id` is indeed no longer `check` (default) but `credit card`?

Reply:
Done. Thanks for the pointer 👍  @KarinG

Reply:
Awesome - it's just a few lines but nice to secure this 👍